src/Features/RemotePaymentMethods/Init.php

```diff
@@ -38,7 +38,15 @@ public static function get_methods() {
 			$methods[] = $method;
 		}
 
-		return $methods;
+		return array_values(
+			array_filter(
+				$methods,
+				function( $method ) {
+					return $method->is_visible;
```

Comment:
Should we check to see if `is_visible` is set and keep the method if not?

Reply:
Good call, done in 06039e81846abd34dacc3bc0bf547f5aedd68f68.